treeherder/settings/base.py

```diff
@@ -195,7 +195,13 @@
     # queue for mirroring the sheriffing activity to tbpl
     Queue('high_priority', Exchange('default'), routing_key='high_priority'),
     Queue('pushlog', Exchange('default'), routing_key='pushlog'),
-    Queue('buildapi', Exchange('default'), routing_key='buildapi')
+    Queue('buildapi', Exchange('default'), routing_key='buildapi'),
+
+    Queue('process_objects', Exchange('default'), routing_key='process_objects'),
+    Queue('cycle_data', Exchange('default'), routing_key='cycle_data'),
+    Queue('calculate_eta', Exchange('default'), routing_key='calculate_eta'),
+    Queue('populate_performance_series', Exchange('default'), routing_key='populate_performance_series'),
```

Comment:
Sorry for not seeing this sooner, but I think we also need to add a routing_key here:
https://github.com/mozilla/treeherder-service/blob/master/treeherder/etl/perf_data_adapters.py#L254